compliance/node/index/indexer.go

```diff
@@ -37,7 +36,7 @@ import (
 const (
 	layerMediaType = "application/vnd.claircore.filesystem"
 
-	rhcosPackageDB = "sqlite:usr/share/rpm"
```

Comment:
Why do we remove the prefix? Is it going to be added later or is no longer needed?

Reply:
It was just an empirical observation by me - with the prefix all packages ended up being filtered out. I'll try to track this down to the code change in claircore to be certain.

Reply:
I realized that the prefix is only stripped in `claircore 1.5.40`. After testing the updated `claircore 1.5.44`, the packageDB is again prefixed with `sqlite:` as before. So I rebased onto the latest version of Ross's PR and reverted the prefix change.
```